background.js

```diff
@@ -1,24 +1,27 @@
 function onBeforeRequest(data){
     //perhaps put 'data' in parameter
     console.log("top of on before request method");
-    console.log(data);
+    console.log(data.initiator);
```

Comment:
This isn't quite the property that we care about. The `initiator` is "the origin where the request was initiated" (https://developer.chrome.com/extensions/webRequest#event-onBeforeRequest), but it's not the url of the request itself. I think you can guess which property on `data` we need to get the url :) 

Once you've figured out which property you need, try logging that one out to see how many different tracker/other domains a page like cnn or tmz hits.